spock-specs/src/test/groovy/org/spockframework/mock/AccessProtectedPropsSpec.groovy

```diff
@@ -0,0 +1,81 @@
+package org.spockframework.mock
+
+import org.spockframework.runtime.model.parallel.ExecutionMode
+import spock.lang.Execution
+import spock.lang.Issue
+import spock.lang.Specification
+
+@Execution(ExecutionMode.SAME_THREAD)
```

Comment:
I have added this to ensure execution order, because if the test `"Access protected fields via access methods without spy"` runs before the `"Access protected const should be accessible in Groovy 3&4 Issue #1501"` the second test would not fail without the fix, because the MetaClass in Groovy was already initialized and pointing to the correct property. Shall I add a comment to that to clarifiy this.

Reply:
If the order is important then `@Stepwise` seems to be the better choice.